docs/usage/configuration-options.md

```diff
@@ -2445,6 +2445,10 @@ Configure this to `true` if you wish to receive one PR for every separate major
 e.g. if you are on webpack@v1 currently then default behavior is a PR for upgrading to webpack@v3 and not for webpack@v2.
 If this setting is true then you would get one PR for webpack@v2 and one for webpack@v3.
 
+## separateMultipleMinor
+
+Configure this to `true` if you wish to receive one PR for every available minor version upgrade of a dependency for each major upgrade.
```

Comment:
This needs more explanation than just one hard to understand sentence.
Can you make the behavior visual, by making a list or table?

I'm thinking of something like this:

> If you enable `separateMultipleMinor` and you have a package `foo` with versions `x.y.z`, `x.y.z` and `x.y.z` then you'll get update PRs for these versions:
> 
> - List version 1
> - List version 2

Reply:
I agree, however I didn’t want to put too much effort into the docs until the premise behind what I attempted was agreed.

Reply:
That's okay! I understand wanting to know if the idea is good or not before putting effort into the docs. 😄 

Please expand the documentation once you have approval for your code. 😉